scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,118 @@
+      --set central.resources.limits.cpu=1 \
+      --set scanner.autoscaling.disable=true \
+      --set scanner.replicas=1 \
+      --set scanner.resources.requests.memory=500Mi \
+      --set scanner.resources.requests.cpu=500m \
+      --set scanner.resources.limits.memory=2500Mi \
+      --set scanner.resources.limits.cpu=2000m
+fi
+
+kubectl -n stackrox rollout status deploy/central --timeout=3m
+
+echo "Forwarding ports to central"
+
+kubectl -n stackrox port-forward deploy/central --pod-running-timeout=1m0s 8000:8443 > /dev/null 2>&1 &
+
+echo "Generating an init bundle with shared secrets"
+
+kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
+  --password "${STACKROX_ADMIN_PASSWORD}" \
+  central init-bundles generate stackrox-init-bundle --output - 1> stackrox-init-bundle.yaml
+
+
+if $SMALL_INSTALL; then
+    echo "Installing the first small secured cluster"
+    helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services \
+        -f stackrox-init-bundle.yaml \
+        --set clusterName="First-Secured-Cluster" \
+        --set sensor.resources.requests.memory=500Mi \
+        --set sensor.resources.requests.cpu=500m \
+        --set sensor.resources.limits.memory=500Mi \
+        --set sensor.resources.limits.cpu=500m
+else
+    echo "Installing the first secured cluster"
+    helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services \
+        -f stackrox-init-bundle.yaml \
+        --set clusterName="First-Secured-Cluster"
+fi
+
+echo "Logging into StackRox in the browser"
+
+curl_status=0
+target_url="$(curl -sSkf -u "admin:${STACKROX_ADMIN_PASSWORD}" -o /dev/null -w '%{redirect_url}' "https://localhost:8000/sso/providers/basic/4df1b98c-24ed-4073-a9ad-356aec6bb62d/challenge?micro_ts=0")" || curl_status=$?
+if [[ "$curl_status" -eq 0 && -n "$target_url" ]]; then
+    if [[ -n "$target_url" ]]; then
+        auth_error="$(sed -E '/^.*#error=([^&]+)(&.*)?$/!d;s//\1/' <<<"$target_url" | tr '+' ' ')"
+        if [[ -z "$auth_error" ]]; then
+            echo "Logging you in via ${target_url} ..."
+            platform="$(uname)"
+            if [[ "$platform" == "Linux" ]]; then
+                xdg-open "${target_url}" >/dev/null &
+                echo ""
+                echo "You may access the dashboard via https://localhost:8000/main/dashboard now."
+            elif [[ "$platform" == "Darwin" ]]; then
+                open "${target_url}" &
+                echo ""
+                echo "You may access the dashboard via https://localhost:8000/main/dashboard now."
+            else
+                echo "Unsupported platform '$platform', please open $@ in a browser"
+            fi
+        else
+            echo "Authentication error: ${auth_error}"
+        fi
+    else
+        echo "UNEXPECTED: Could not determine target URL"
+    fi
+else
+    echo "Could not determine login URL for basic auth user"
```

Comment:
1. [minor] Could you please `echo` these to `stderr` because these seem like errors?
2. [nit] Could you please think how to reduce the level of nestedness of the code? It is a bit hard to parse which `else` belongs to which condition even despite indentation.
I don't have a suggestion on (2) yet.

Reply:
1. If I do 1. would you also say that the script should exit with an error in those cases? Because in that case
2. number 2) would be trivial since if I exit the script on failure to retrieve the link I would not need the nesting. I don't think that failing to retrieve that link is necessarily an indication that the installation failed though. Alternatively I could package some of the nested ifs into function calls since that might look nicer, but it also seems like questionable design to me if they would only get called once each. Thoughts?

Reply:
Either way I've done 1. for now, as for 2. if you think it's better style I'll change it to use function, otherwise it will stay as is.

Reply:
1 looks ok. Continuing 2 in https://github.com/stackrox/stackrox/pull/2565#discussion_r954256314